src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2307,6 +2308,18 @@ bool os::Linux::print_container_info(outputStream* st) {
     st->print_cr("%s", j == OSCONTAINER_ERROR ? "not supported" : "unlimited");
   }
 
+  j = OSContainer::OSContainer::pids_max();
+  st->print("maximum number of tasks: ");
+  if (j > 0) {
+    st->print_cr(JLONG_FORMAT, j);
+  } else {
+    if (j == -3) {
+      st->print_cr("max");
+    } else {
+      st->print_cr("%s", j == OSCONTAINER_ERROR ? "not supported" : "unlimited");
```

Comment:
We should treat the unlimited case similar to how we handle them elsewhere. I'm not sure this magic constant of `-3` gives us any more info that we'd get with `-1` that we use elsewhere.